ant-chapters/chapter1.tex

```diff
@@ -1,5 +1,5 @@
 \chapter{Rings of integers}\llabel{ring-of-integers}
-When we have a field extension $L$ of $\Q$, we would like to define a ring of integers for $L$, with properties similar to the ring $\Z\subeq \Q$. We will define this ring of integers in a slightly more general context.

```

Comment:
No more long introduction in this chapter so no reason to remove it now, but I plan to send some changes to the integrality section in a near future where I rewrite this sentence anyway.